spec/venue_spec.rb

```diff
@@ -53,3 +46,38 @@
     end
   end
 end
+
+  describe '#venue not over capacity' do
+    it 'returns true if venue over capacity'do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      expect(venue.over_capacity?).to eq(false)
+    end
+  end
+
+  describe '#venue over capacity' do
+    it 'returns true if venue over capacity'do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Kat')
+      expect(venue.over_capacity?).to eq(true)
+    end
+  end
+
+  describe '#venue over capacity' do
+    it 'kicks out most recent over-capacity patrons' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Kat')
+      expect(venue.kick_out).to eq(['Mike', 'Megan', 'Bob', 'James'])
```

Comment:
Nice! Looks like you went above and beyond to make sure the patrons you were expecting to be kicked out were kicked out.